src/resources/auth/auth.repository.js

```diff
@@ -0,0 +1,37 @@
+const bcrypt = require('bcrypt');
+const { users } = require('../../loader/dbconnect');
+// const HttpError = require('../../tools/httpError');
```

Comment:
if you don't use this then please remove it

Reply:
fixed